packages/client/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "@babel/eslint-parser": "^7.19.1",
```

Comment:
Maybe we should create a separate PR with these changes related to eslint so it's clear what is connected to the feature and what's not

Reply:
I don't know why this part is not included in the previous fixes, but that's a good suggestion. Will do! Thanks!